### PR TITLE
Add 'beta' VPill to Audio in the content switcher

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Related #631 by @panchovm

## Description

This PR adds constants for media type statuses (`supported` for images and all media, `beta` for audio, and `additional` for video), and displays the `BETA` `VPill` to the right of Audio in content switcher dropdown (popover, modal).

## Testing Instructions

Run the app and click on the content switcher button in the header. You should see a yellow 'Beta' pill to the right of 'Audio'.

This PR is based on the 'add/pill-component' branch, so it contains a lot of unrelated changes, and I will clean it up after that PR is merged.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by panchovm** [comment]
The content switcher item has inconsistent paddings on the left and right. I noticed the SVG element has margins that make the element bigger. The component has new spacings to make the switcher popover less intrusive, check the #639 ticket.

On the other hand, I do not see the hover style of the other elements.

### Screenshot
CleanShot 2022-01-19 at 10 17 37@2x

**Comment by zackkrida** [comment]
Looks like the margin issues are the main thing that need to be fixed. Once the sizing is correct this will be a quick approval!

**Comment by dhruvkb** [comment]
I worked on clearing the merge conflicts for the PR [here](https://github.com/WordPress/openverse-frontend/compare/main...add/audio_pill_updated?expand=1). @obulat do take a look, and if all's well, update this PR to bring it to mergeable status.

**Comment by obulat** [comment]
@panchovm , could you please look at the latest version:
Screen Shot 2022-01-21 at 12 46 25 PM
There is about 2px of safe area around the svg icon itself, and then there is 8 px of margin inside the gray area, and 8px of outer padding from the Item Group.

**Comment by panchovm** [comment]
I'm still seeing inconsistent spacings. Here is attached a gif from the Releases file showing the internal spacings of the popover.

**Comment by obulat** [comment]
### Mobile switcher

_(tabbing currently doesn't work, so you can only see the pink outline on focus by manually setting `focus-visible` in developer tools)_:
Screen Shot 2022-01-21 at 9 09 12 PM
All content is hovered, and images is `focused` using tab (simulated, as real tab doesn't work)

###  Desktop switcher
content-switcher

File: src/components/VContentSwitcher/VContentItem.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <VItem
+    :selected="selected"
+    :is-first="itemId === 0"
+    @click.native="$emit('click', item)"
+  >
+    <div class="flex flex-row items-center py-3">
```
Comment (zackkrida) [inline src/components/VContentSwitcher/VContentItem.vue:7]:
```suggestion
    <div class="flex flex-row items-center py-3">
```

The `py-3` seems like it is making each item too tall, which is the main problem i notice when testing.

**Comment by obulat** [comment]
Setting it to py-2 seems to fix it:
content-switcher

**Review by zackkrida — APPROVED** [review]
Added the beta pill, and it looks fine ✅

**Pull request merged**